ui/webui/src/components/installation/InstallationProgress.jsx

```diff
@@ -81,26 +83,43 @@ export class InstallationProgress extends React.Component {
 
         if (steps === undefined) { return null }
 
+        let icon;
+        let title;
+        if (status === "success") {
+            icon = CheckCircleIcon;
+            title = _("Installed");
+        } else if (status === "danger") {
+            icon = ExclamationCircleIcon;
+            title = _(_("Installation failed"));
+        } else {
+            title = _("Installing");
+        }
+
         return (
-            <Bullseye>
-                <EmptyState variant="large">
-                    <EmptyStateBody>
-                        <Progress
-                          id="installation-progress"
-                          label={cockpit.format("$0 of $1", step, steps)}
-                          max={steps}
-                          min={0}
-                          title={message}
-                          value={step}
-                          valueText={cockpit.format("$0 of $1", step, steps)}
-                          variant={status}
-                        />
-                    </EmptyStateBody>
-                    {status === "success" &&
-                    <EmptyStateSecondaryActions>
-                        <Button onClick={exitGui}>{_("Reboot")}</Button>
-                    </EmptyStateSecondaryActions>}
-                </EmptyState>
+            <Bullseye className={"installation-progress-status-" + status}>
+                <EmptyStatePanel
+                  icon={icon}
+                  loading={!icon}
+                  paragraph={
+                      status !== "success"
+                          ? <Progress
+                              id="installation-progress"
+                              label={cockpit.format("$0 of $1", step, steps)}
+                              max={steps}
+                              min={0}
+                              title={message}
+                              value={step}
+                              valueText={cockpit.format("$0 of $1", step, steps)}
+                              variant={status}
+                          />
+                          : <Text>{_("Anaconda is successfully installed. To begin using it, reboot your system.")}</Text>
```

Comment:
I think there should be "Fedora" instead of Anaconda. :)

Reply:
Omg yes, I should have noticed.